CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Allow usage of nested arrays to be used in insert/update statements
```

Comment:
I wonder if this should be phrased as fix for geo_point arrays. 
It is afaik not possible to create a table with nested arrays ( `create table t (x array(array(int)))` results in an error) so this changes somehow implies that it is now possible to use something that is not possible